From 270423008d401d9f03a31f08c53a7b13fa193573 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Wed, 23 Jun 2021 09:23:31 +0200
Subject: [PATCH] Fix test_unmodel to accept ujson encoding without whitespace

The ujson encoding for test_unmodel data is b'"Conventions":"UGRID-0.9.0"'
(without a space after ':') that breaks the test for me.  Update
the test to make the whitespace optional.

That said, I have tried multiple ujson versions, going back to <3,
and was not able to reproduce the output with space.
---
 fsspec/implementations/tests/test_reference.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/fsspec/implementations/tests/test_reference.py b/fsspec/implementations/tests/test_reference.py
index a1a8fdc..15a32d5 100644
--- a/fsspec/implementations/tests/test_reference.py
+++ b/fsspec/implementations/tests/test_reference.py
@@ -104,7 +104,8 @@ jdata = """{
 
 def test_unmodel():
     refs = _unmodel_hdf5(json.loads(jdata))
-    assert b'"Conventions": "UGRID-0.9.0"' in refs[".zattrs"]
+    # apparently the output may or may not contain a space after ':'
+    assert b'"Conventions":"UGRID-0.9.0"' in refs[".zattrs"].replace(b' ', b'')
     assert refs["adcirc_mesh/0"] == ("https://url", 8928, 8932)
 
 
-- 
2.32.0

